src/functions/template-tags/general.php

```diff
@@ -1518,48 +1527,60 @@ function tribe_events_get_the_excerpt( $post = null, $allowed_html = null, $skip
 		 */
 		$remove_shortcodes = apply_filters( 'tribe_events_excerpt_shortcode_removal', true );
 
-		// Get the Excerpt or content based on what is available
-		$excerpt = has_excerpt( $post->ID ) ? $post->post_excerpt : $post->post_content;
+		$cache_excerpts_key = implode( ':', [
+			$post->ID,
+			$skip_postdata_manipulation,
+			$allow_shortcodes,
+			$remove_shortcodes,
+			json_encode( $allowed_html )
+		] );
+
+		if ( ! isset( $cache_excerpts[ $cache_excerpts_key ] ) ) {
+			// Get the Excerpt or content based on what is available
+			$excerpt = has_excerpt( $post->ID ) ? $post->post_excerpt : $post->post_content;
+
+			// If shortcode filter is enabled let's process them
```

Comment:
:no_entry_sign: Inline comments must end in full-stops, exclamation marks, or question marks

<sub>hash: b61eb03058605f0e08ecca5e60dc4a1e</sub>